android/java/org/chromium/chrome/browser/crypto_wallet/activities/NftDetailActivity.java

```diff
@@ -113,13 +123,22 @@ protected void triggerLayoutInflation() {
         mNetworkNameView = findViewById(R.id.blockchain_content);
 
         mTokenStandardView = findViewById(R.id.token_standard_content);
-        mTokenStandardView.setText(R.string.brave_wallet_nft_erc_721);
 
-        mTokenIdView = findViewById(R.id.token_id_content);
+        if (mIsErc721) {
+            mTokenStandardView.setText(R.string.brave_wallet_nft_erc_721);
+        } else {
+            mTokenStandardView.setText(R.string.brave_wallet_nft_sol_spl);
+        }
```

Comment:
Since we are showing other NFTs so this will show other nfts with SOL icon. Better to check sol type `isNft` and use a generic icon for the else case

Reply:
Icon? What do you mean? I'm not able to find an icon on this screen.

Reply:
I meant the text, overlooked.